### PR TITLE
Adding support for Storage pool enhancements

### PR DESCRIPTION
**Pull request by trisha-dell**
# Description
Adding support for parameters for Storage pool.

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [X] I have performed Ansible Sanity test using --docker default
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [X] I have maintained at least 90% code coverage
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Functional testing
- [X] Unit testing

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: roles/powerflex_sdc/tasks/install_sdc.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@
     mode: "0600"
     owner: "root"
     group: "root"
-  notify: restart scini
+  notify:
+    - restart scini
```
Comment (felixs88) [inline roles/powerflex_sdc/tasks/install_sdc.yml:74]:
```suggestion
    - Restart scini
```

File: roles/powerflex_gateway/tasks/install_keepalived.yml [hunk 2]
```diff
@@ -25,4 +25,5 @@
     mode: '0600'
     owner: root
     group: root
-  notify: restart keepalived
+  notify:
+    - restart keepalived
```
Comment (felixs88) [inline roles/powerflex_gateway/tasks/install_keepalived.yml:29]:
```suggestion
    - Restart keepalived
```

**Review by meenakshidembi691 — APPROVED** [review]

**Comment by sachin-apa** [comment]
run e2e

File: plugins/modules/storagepool.py [hunk 3]
```diff
@@ -648,7 +777,7 @@ def get_storage_pool(self, storage_pool_id=None, storage_pool_name=None,
                 filter_fields.update({'protectionDomainId': pd_id})
             pool_details = self.powerflex_conn.storage_pool.get(
                 filter_fields=filter_fields)
-            if pool_details:
+            if pool_details != []:
                 if len(pool_details) > 1:
```
[the remark this answers is not in the document]
Comment (trisha-dell) [reply plugins/modules/storagepool.py:782]: there is an else block as well, hence can not be clubbed.

**Review by felixs88 — APPROVED** [review]

**Review by Bhavneet-Sharma — APPROVED** [review]

**Pull request merged**